docs/restic.md

```diff
@@ -108,21 +112,22 @@ You're now ready to use Ark with restic.
     kubectl -n foo annotate pod/sample backup.ark.heptio.com/backup-volumes=pvc-volume,emptydir-volume
     ```
 
-
-    This annotation can also be provided in the pod template spec if using a deployment, daemonset, etc.
+    This annotation can also be provided in the pod template spec if you're using a deployment or daemonset
     to manage your pods.
 
-1. Take an Ark backup as usual:
+1. Take an Ark backup:
+
     ```bash
     ark backup create NAME OPTIONS...
     ```
 
-1. When the backup has completed, view information about your pod volume backups:
+1. When the backup completes, view information about the backups:
+
     ```bash
     kubectl -n heptio-ark get podvolumebackups -l ark.heptio.com/backup-name=YOUR_BACKUP_NAME -o yaml
```

Comment:
@skriss this needs to be updated now that #550 is in

Reply:
Steve tells me it's in #643